canonical-web-and-design/snapcraft.io
# Remove unused JS on blog pages

## Done

Removed all unused JS on blog pages

## Issue / Card

(Partly) Fixes #3262

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/blog
- Check that the newsletter form works
- Click through to a blog post
- Check that the newsletter form works
